### ISSUE TITLE
Found this error frequently on latest release build

### ISSUE THREAD

**Issue by yashkrishnan** [Ashish-Bansal/OneTapVideoDownload#1]
```
java.lang.NoClassDefFoundError: okhttp3.internal.tls.OkHostnameVerifier
at okhttp3.OkHttpClient$Builder.<init>(OkHttpClient.java:364)
at okhttp3.OkHttpClient.<init>(OkHttpClient.java:151)
at com.phantom.onetapvideodownload.downloader.DownloadHandler.handleRemoteFileDownload(DownloadHandler.java:107)
at com.phantom.onetapvideodownload.downloader.DownloadHandler.downloadFile(DownloadHandler.java:75)
at com.phantom.onetapvideodownload.downloader.DownloadHandler.startDownload(DownloadHandler.java:57)
at com.phantom.onetapvideodownload.downloader.DownloadManager.handleActionResumeDownload(DownloadManager.java:302)
at com.phantom.onetapvideodownload.downloader.DownloadManager.onStartCommand(DownloadManager.java:182)
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2899)
at android.app.ActivityThread.access$2100(ActivityThread.java:154)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1445)
at android.os.Handler.dispatchMessage(Handler.java:110)
at android.os.Looper.loop(Looper.java:193)
at android.app.ActivityThread.main(ActivityThread.java:5334)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:515)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:835)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:651)
at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
at dalvik.system.NativeStart.main(Native Method)
```

**Comment by Ashish-Bansal** [comment]
@yashwanth-krishnan 

Hi Yashwanth,

This bug should be solved in latest release i.e. version 3.2. Can you please try that version ? 

Regards,
Ashish Bansal

**Comment by Ashish-Bansal** [comment]
Fixed with @3d405bacf869ee10735e6b85dc1d78dba9c3ef36

**Comment by yashkrishnan** [comment]
@Ashish-Bansal 
Hi Ashish,

As I tested, it seems like that bug has got fixed.

Great job,
Yashwanth Krishnan
